### ISSUE TITLE
Feed generation fails

### ISSUE THREAD

**Issue by stephanschubert** [ChristopherBiscardi/gatsby-mdx#249]
**Describe the bug**

Cannot generate feed from MDX files using React components. (seems to be 2 bugs at once)

**To Reproduce**

```js
    {
      resolve: "gatsby-mdx",
      options: {
        extensions: [".mdx", ".md"],
        defaultLayouts: {
          default: require.resolve("./src/templates/post.js"),
        },
        globalScope: `
          import Notes from "components/notes";

          export default {
            Notes
          };
        `,
      },
    },
    {
      options: {
        ...require("gatsby-mdx/feed"),
      },
      resolve: `gatsby-plugin-feed`,
    },
```

```
ReferenceError: __WEBPACK_EXTERNAL_MODULE_react__ is not defined
	at Object.react (evalmachine.<anonymous>:55240:18)
	at __webpack_require__ (evalmachine.<anonymous>:20:30)
	at Module.../../node_modules/gatsby-mdx/utils/wrap-root-render-html-entry.js (evalmachine.<anonymous>:2582:63)
	at __webpack_require__ (evalmachine.<anonymous>:20:30)
	at ../../node_modules/@babel/runtime/helpers/arrayWithoutHoles.js.module.exports (evalmachine.<anonymous>:84:18)
	at evalmachine.<anonymous>:87:10
	at Script.runInContext (vm.js:102:20)
	at Script.runInNewContext (vm.js:108:17)
	at module.exports (/Users/sschubert/git/hydra/node_modules/eval/eval.js:69:12)
	at /Users/sschubert/git/hydra/node_modules/static-site-generator-webpack-plugin/index.js:44:22' ]

error UNHANDLED EXCEPTION


  Error: no such file or directory

  - MemoryFileSystem.js:114 MemoryFileSystem.readFileSync
    [hydra]/[memory-fs]/lib/MemoryFileSystem.js:114:10

  - render-html.js:56 compiler.run
    [hydra]/[gatsby-mdx]/utils/render-html.js:56:26

  ...
```

1. React not found—this could be related to the (incomplete) `globalScope` feature?

2. `render-html.js` [attempts to read `/index.html`](https://github.com/ChristopherBiscardi/gatsby-mdx/blob/master/packages/gatsby-mdx/utils/render-html.js#L54) but there's only an `/output.txt`?

**Expected behavior**

Feed generation completes without errors.

**Comment by ChristopherBiscardi** [comment]
The default feed.js file in MDX was recently swapped over to using `html` and not `excerpt`.

For information on `2` and why we're looking for `index.html`, you may want to [read this blog post](https://www.christopherbiscardi.com/post/contorting-webpack-to-render-html-for-gatsby-mdx) I wrote on the subject. Basically, you're looking at the most convoluted file in gatsby-mdx and it requires doing some weird stuff.

`1` definitely looks like a bug.

**Comment by stephanschubert** [comment]
@ChristopherBiscardi The error about missing `React` is gone now but the components used aren't rendered:

```xml
// from public/rss.xml
<item>...<description><![CDATA[<Notes.Experiment /> ...
```

The JSX representation is still in place?

**Comment by ChristopherBiscardi** [comment]
@stephanschubert hmm, I'm not seeing the same result on the [kitchen-sink rss feed](https://gatsby-mdx-kitchen-sink.netlify.com/rss.xml] for [this page](https://github.com/ChristopherBiscardi/gatsby-mdx/blob/edafe526c5a9d124f4f8db68932967f3a2377b66/examples/kitchen-sink/content/non-page-global-scope.mdx) that uses globalScope and components.

The color picker renders weirdly in feedly, but I think that's because it's mostly gradients and  feedly's algorithm strips that stuff.

<details>
<summary>Color Picker Page RSS output</summary>

```
<item>
<title>
<![CDATA[ No title ]]>
</title>
<description>
<![CDATA[
A non-page In not the pages directory, with nothing special about it
]]>
</description>
<link>
gatsby-mdx-kitchen-sink.netlify.com/posts/non-page-global-scope
</link>
<guid isPermaLink="false">
gatsby-mdx-kitchen-sink.netlify.com/posts/non-page-global-scope
</guid>
<content:encoded>
<div><h1>A non-page</h1><p>In not the pages directory, with nothing special about it</p><div style="width:200px;padding:10px 10px 0;box-sizing:initial;background:#fff;-ms-border-radius:4px;-moz-border-radius:4px;-o-border-radius:4px;-webkit-border-radius:4px;border-radius:4px;-ms-box-shadow:0 0 0 1px rgba(0,0,0,.15), 0 8px 16px rgba(0,0,0,.15);-moz-box-shadow:0 0 0 1px rgba(0,0,0,.15), 0 8px 16px rgba(0,0,0,.15);-o-box-shadow:0 0 0 1px rgba(0,0,0,.15), 0 8px 16px rgba(0,0,0,.15);-webkit-box-shadow:0 0 0 1px rgba(0,0,0,.15), 0 8px 16px rgba(0,0,0,.15);box-shadow:0 0 0 1px rgba(0,0,0,.15), 0 8px 16px rgba(0,0,0,.15)" class="sketch-picker "><div style="width:100%;padding-bottom:75%;position:relative;overflow:hidden"><div style="position:absolute;top:0px;right:0px;bottom:0px;left:0px;background:hsl(249.99999999999994,100%, 50%)"><style> .saturation-white { background: -webkit-linear-gradient(to right, #fff, rgba(255,255,255,0)); background: linear-gradient(to right, #fff, rgba(255,255,255,0)); } .saturation-black { background: -webkit-linear-gradient(to top, #000, rgba(0,0,0,0)); background: linear-gradient(to top, #000, rgba(0,0,0,0)); } </style><div style="position:absolute;top:0px;right:0px;bottom:0px;left:0px" class="saturation-white"><div style="position:absolute;top:0px;right:0px;bottom:0px;left:0px" class="saturation-black"></div><div style="position:absolute;top:70%;left:66.66666666666667%;cursor:default"><div style="width:4px;height:4px;-ms-box-shadow:0 0 0 1.5px #fff, inset 0 0 1px 1px rgba(0,0,0,.3), 0 0 1px 2px rgba(0,0,0,.4);-moz-box-shadow:0 0 0 1.5px #fff, inset 0 0 1px 1px rgba(0,0,0,.3), 0 0 1px 2px rgba(0,0,0,.4);-o-box-shadow:0 0 0 1.5px #fff, inset 0 0 1px 1px rgba(0,0,0,.3), 0 0 1px 2px rgba(0,0,0,.4);-webkit-box-shadow:0 0 0 1.5px #fff, inset 0 0 1px 1px rgba(0,0,0,.3), 0 0 1px 2px rgba(0,0,0,.4);box-shadow:0 0 0 1.5px #fff, inset 0 0 1px 1px rgba(0,0,0,.3), 0 0 1px 2px rgba(0,0,0,.4);-ms-border-radius:50%;-moz-border-radius:50%;-o-border-radius:50%;-webkit-border-radius:50%;border-radius:50%;cursor:hand;-ms-transform:translate(-2px, -2px);-moz-transform:translate(-2px, -2px);-o-transform:translate(-2px, -2px);-webkit-transform:translate(-2px, -2px);transform:translate(-2px, -2px)"></div></div></div></div></div><div style="display:flex" class="flexbox-fix"><div style="padding:4px 0;-webkit-box-flex:1;-moz-box-flex:1;-webkit-flex:1;-ms-flex:1;flex:1"><div style="position:relative;height:10px;overflow:hidden"><div style="position:absolute;top:0px;right:0px;bottom:0px;left:0px"><div class="hue-horizontal" style="padding:0 2px;position:relative;height:100%"><style> .hue-horizontal { background: linear-gradient(to right, #f00 0%, #ff0 17%, #0f0 33%, #0ff 50%, #00f 67%, #f0f 83%, #f00 100%); background: -webkit-linear-gradient(to right, #f00 0%, #ff0 17%, #0f0 33%, #0ff 50%, #00f 67%, #f0f 83%, #f00 100%); } .hue-vertical { background: linear-gradient(to top, #f00 0%, #ff0 17%, #0f0 33%, #0ff 50%, #00f 67%, #f0f 83%, #f00 100%); background: -webkit-linear-gradient(to top, #f00 0%, #ff0 17%, #0f0 33%, #0ff 50%, #00f 67%, #f0f 83%, #f00 100%); } </style><div style="position:absolute;left:69.44444444444443%"><div style="margin-top:1px;width:4px;-ms-border-radius:1px;-moz-border-radius:1px;-o-border-radius:1px;-webkit-border-radius:1px;border-radius:1px;height:8px;-ms-box-shadow:0 0 2px rgba(0, 0, 0, .6);-moz-box-shadow:0 0 2px rgba(0, 0, 0, .6);-o-box-shadow:0 0 2px rgba(0, 0, 0, .6);-webkit-box-shadow:0 0 2px rgba(0, 0, 0, .6);box-shadow:0 0 2px rgba(0, 0, 0, .6);background:#fff;-ms-transform:translateX(-2px);-moz-transform:translateX(-2px);-o-transform:translateX(-2px);-webkit-transform:translateX(-2px);transform:translateX(-2px)"></div></div></div></div></div><div style="position:relative;height:10px;margin-top:4px;overflow:hidden"><div style="position:absolute;top:0px;right:0px;bottom:0px;left:0px"><div style="position:absolute;top:0px;right:0px;bottom:0px;left:0px;overflow:hidden"><div style="position:absolute;top:0px;right:0px;bottom:0px;left:0px;background:url(null) center left"></div></div><div style="position:absolute;top:0px;right:0px;bottom:0px;left:0px;background:linear-gradient(to right, rgba(34,25,77, 0) 0%, rgba(34,25,77, 1) 100%)"></div><div style="position:relative;height:100%;margin:0 3px"><div style="position:absolute;left:100%"><div style="width:4px;-ms-border-radius:1px;-moz-border-radius:1px;-o-border-radius:1px;-webkit-border-radius:1px;border-radius:1px;height:8px;-ms-box-shadow:0 0 2px rgba(0, 0, 0, .6);-moz-box-shadow:0 0 2px rgba(0, 0, 0, .6);-o-box-shadow:0 0 2px rgba(0, 0, 0, .6);-webkit-box-shadow:0 0 2px rgba(0, 0, 0, .6);box-shadow:0 0 2px rgba(0, 0, 0, .6);background:#fff;margin-top:1px;-ms-transform:translateX(-2px);-moz-transform:translateX(-2px);-o-transform:translateX(-2px);-webkit-transform:translateX(-2px);transform:translateX(-2px)"></div></div></div></div></div></div><div style="width:24px;height:24px;position:relative;margin-top:4px;margin-left:4px;-ms-border-radius:3px;-moz-border-radius:3px;-o-border-radius:3px;-webkit-border-radius:3px;border-radius:3px"><div style="position:absolute;top:0px;right:0px;bottom:0px;left:0px;background:url(null) center left"></div><div style="position:absolute;top:0px;right:0px;bottom:0px;left:0px;-ms-border-radius:2px;-moz-border-radius:2px;-o-border-radius:2px;-webkit-border-radius:2px;border-radius:2px;background:rgba(34,25,77,1);-ms-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15), inset 0 0 4px rgba(0,0,0,.25);-moz-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15), inset 0 0 4px rgba(0,0,0,.25);-o-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15), inset 0 0 4px rgba(0,0,0,.25);-webkit-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15), inset 0 0 4px rgba(0,0,0,.25);box-shadow:inset 0 0 0 1px rgba(0,0,0,.15), inset 0 0 4px rgba(0,0,0,.25)"></div></div></div><div style="display:flex;padding-top:4px" class="flexbox-fix"><div style="-webkit-box-flex:2;-moz-box-flex:2;-webkit-flex:2;-ms-flex:2;flex:2"><div style="position:relative"><input style="width:80%;padding:4px 10% 3px;border:none;-ms-box-shadow:inset 0 0 0 1px #ccc;-moz-box-shadow:inset 0 0 0 1px #ccc;-o-box-shadow:inset 0 0 0 1px #ccc;-webkit-box-shadow:inset 0 0 0 1px #ccc;box-shadow:inset 0 0 0 1px #ccc;font-size:11px" value="22194D" spellcheck="false"/><span style="display:block;text-align:center;font-size:11px;color:#222;padding-top:3px;padding-bottom:4px;text-transform:capitalize">hex</span></div></div><div style="-webkit-box-flex:1;-moz-box-flex:1;-webkit-flex:1;-ms-flex:1;flex:1;padding-left:6px"><div style="position:relative"><input style="width:80%;padding:4px 10% 3px;border:none;-ms-box-shadow:inset 0 0 0 1px #ccc;-moz-box-shadow:inset 0 0 0 1px #ccc;-o-box-shadow:inset 0 0 0 1px #ccc;-webkit-box-shadow:inset 0 0 0 1px #ccc;box-shadow:inset 0 0 0 1px #ccc;font-size:11px" value="34" spellcheck="false"/><span style="display:block;text-align:center;font-size:11px;color:#222;padding-top:3px;padding-bottom:4px;text-transform:capitalize;cursor:ew-resize">r</span></div></div><div style="-webkit-box-flex:1;-moz-box-flex:1;-webkit-flex:1;-ms-flex:1;flex:1;padding-left:6px"><div style="position:relative"><input style="width:80%;padding:4px 10% 3px;border:none;-ms-box-shadow:inset 0 0 0 1px #ccc;-moz-box-shadow:inset 0 0 0 1px #ccc;-o-box-shadow:inset 0 0 0 1px #ccc;-webkit-box-shadow:inset 0 0 0 1px #ccc;box-shadow:inset 0 0 0 1px #ccc;font-size:11px" value="25" spellcheck="false"/><span style="display:block;text-align:center;font-size:11px;color:#222;padding-top:3px;padding-bottom:4px;text-transform:capitalize;cursor:ew-resize">g</span></div></div><div style="-webkit-box-flex:1;-moz-box-flex:1;-webkit-flex:1;-ms-flex:1;flex:1;padding-left:6px"><div style="position:relative"><input style="width:80%;padding:4px 10% 3px;border:none;-ms-box-shadow:inset 0 0 0 1px #ccc;-moz-box-shadow:inset 0 0 0 1px #ccc;-o-box-shadow:inset 0 0 0 1px #ccc;-webkit-box-shadow:inset 0 0 0 1px #ccc;box-shadow:inset 0 0 0 1px #ccc;font-size:11px" value="77" spellcheck="false"/><span style="display:block;text-align:center;font-size:11px;color:#222;padding-top:3px;padding-bottom:4px;text-transform:capitalize;cursor:ew-resize">b</span></div></div><div style="-webkit-box-flex:1;-moz-box-flex:1;-webkit-flex:1;-ms-flex:1;flex:1;padding-left:6px"><div style="position:relative"><input style="width:80%;padding:4px 10% 3px;border:none;-ms-box-shadow:inset 0 0 0 1px #ccc;-moz-box-shadow:inset 0 0 0 1px #ccc;-o-box-shadow:inset 0 0 0 1px #ccc;-webkit-box-shadow:inset 0 0 0 1px #ccc;box-shadow:inset 0 0 0 1px #ccc;font-size:11px" value="100" spellcheck="false"/><span style="display:block;text-align:center;font-size:11px;color:#222;padding-top:3px;padding-bottom:4px;text-transform:capitalize;cursor:ew-resize">a</span></div></div></div><div style="margin:0 -10px;padding:10px 0 0 10px;border-top:1px solid #eee;display:flex;flex-wrap:wrap;position:relative" class="flexbox-fix"><div style="width:16px;height:16px;margin:0 10px 10px 0"><span><div style="background:#D0021B;height:100%;width:100%;cursor:pointer;position:relative;outline:none;-ms-border-radius:3px;-moz-border-radius:3px;-o-border-radius:3px;-webkit-border-radius:3px;border-radius:3px;-ms-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-moz-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-o-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-webkit-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);box-shadow:inset 0 0 0 1px rgba(0,0,0,.15)" title="#D0021B" tabindex="0"></div></span></div><div style="width:16px;height:16px;margin:0 10px 10px 0"><span><div style="background:#F5A623;height:100%;width:100%;cursor:pointer;position:relative;outline:none;-ms-border-radius:3px;-moz-border-radius:3px;-o-border-radius:3px;-webkit-border-radius:3px;border-radius:3px;-ms-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-moz-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-o-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-webkit-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);box-shadow:inset 0 0 0 1px rgba(0,0,0,.15)" title="#F5A623" tabindex="0"></div></span></div><div style="width:16px;height:16px;margin:0 10px 10px 0"><span><div style="background:#F8E71C;height:100%;width:100%;cursor:pointer;position:relative;outline:none;-ms-border-radius:3px;-moz-border-radius:3px;-o-border-radius:3px;-webkit-border-radius:3px;border-radius:3px;-ms-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-moz-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-o-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-webkit-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);box-shadow:inset 0 0 0 1px rgba(0,0,0,.15)" title="#F8E71C" tabindex="0"></div></span></div><div style="width:16px;height:16px;margin:0 10px 10px 0"><span><div style="background:#8B572A;height:100%;width:100%;cursor:pointer;position:relative;outline:none;-ms-border-radius:3px;-moz-border-radius:3px;-o-border-radius:3px;-webkit-border-radius:3px;border-radius:3px;-ms-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-moz-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-o-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-webkit-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);box-shadow:inset 0 0 0 1px rgba(0,0,0,.15)" title="#8B572A" tabindex="0"></div></span></div><div style="width:16px;height:16px;margin:0 10px 10px 0"><span><div style="background:#7ED321;height:100%;width:100%;cursor:pointer;position:relative;outline:none;-ms-border-radius:3px;-moz-border-radius:3px;-o-border-radius:3px;-webkit-border-radius:3px;border-radius:3px;-ms-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-moz-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-o-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-webkit-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);box-shadow:inset 0 0 0 1px rgba(0,0,0,.15)" title="#7ED321" tabindex="0"></div></span></div><div style="width:16px;height:16px;margin:0 10px 10px 0"><span><div style="background:#417505;height:100%;width:100%;cursor:pointer;position:relative;outline:none;-ms-border-radius:3px;-moz-border-radius:3px;-o-border-radius:3px;-webkit-border-radius:3px;border-radius:3px;-ms-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-moz-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-o-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-webkit-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);box-shadow:inset 0 0 0 1px rgba(0,0,0,.15)" title="#417505" tabindex="0"></div></span></div><div style="width:16px;height:16px;margin:0 10px 10px 0"><span><div style="background:#BD10E0;height:100%;width:100%;cursor:pointer;position:relative;outline:none;-ms-border-radius:3px;-moz-border-radius:3px;-o-border-radius:3px;-webkit-border-radius:3px;border-radius:3px;-ms-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-moz-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-o-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-webkit-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);box-shadow:inset 0 0 0 1px rgba(0,0,0,.15)" title="#BD10E0" tabindex="0"></div></span></div><div style="width:16px;height:16px;margin:0 10px 10px 0"><span><div style="background:#9013FE;height:100%;width:100%;cursor:pointer;position:relative;outline:none;-ms-border-radius:3px;-moz-border-radius:3px;-o-border-radius:3px;-webkit-border-radius:3px;border-radius:3px;-ms-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-moz-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-o-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-webkit-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);box-shadow:inset 0 0 0 1px rgba(0,0,0,.15)" title="#9013FE" tabindex="0"></div></span></div><div style="width:16px;height:16px;margin:0 10px 10px 0"><span><div style="background:#4A90E2;height:100%;width:100%;cursor:pointer;position:relative;outline:none;-ms-border-radius:3px;-moz-border-radius:3px;-o-border-radius:3px;-webkit-border-radius:3px;border-radius:3px;-ms-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-moz-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-o-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-webkit-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);box-shadow:inset 0 0 0 1px rgba(0,0,0,.15)" title="#4A90E2" tabindex="0"></div></span></div><div style="width:16px;height:16px;margin:0 10px 10px 0"><span><div style="background:#50E3C2;height:100%;width:100%;cursor:pointer;position:relative;outline:none;-ms-border-radius:3px;-moz-border-radius:3px;-o-border-radius:3px;-webkit-border-radius:3px;border-radius:3px;-ms-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-moz-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-o-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-webkit-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);box-shadow:inset 0 0 0 1px rgba(0,0,0,.15)" title="#50E3C2" tabindex="0"></div></span></div><div style="width:16px;height:16px;margin:0 10px 10px 0"><span><div style="background:#B8E986;height:100%;width:100%;cursor:pointer;position:relative;outline:none;-ms-border-radius:3px;-moz-border-radius:3px;-o-border-radius:3px;-webkit-border-radius:3px;border-radius:3px;-ms-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-moz-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-o-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-webkit-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);box-shadow:inset 0 0 0 1px rgba(0,0,0,.15)" title="#B8E986" tabindex="0"></div></span></div><div style="width:16px;height:16px;margin:0 10px 10px 0"><span><div style="background:#000000;height:100%;width:100%;cursor:pointer;position:relative;outline:none;-ms-border-radius:3px;-moz-border-radius:3px;-o-border-radius:3px;-webkit-border-radius:3px;border-radius:3px;-ms-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-moz-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-o-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-webkit-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);box-shadow:inset 0 0 0 1px rgba(0,0,0,.15)" title="#000000" tabindex="0"></div></span></div><div style="width:16px;height:16px;margin:0 10px 10px 0"><span><div style="background:#4A4A4A;height:100%;width:100%;cursor:pointer;position:relative;outline:none;-ms-border-radius:3px;-moz-border-radius:3px;-o-border-radius:3px;-webkit-border-radius:3px;border-radius:3px;-ms-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-moz-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-o-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-webkit-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);box-shadow:inset 0 0 0 1px rgba(0,0,0,.15)" title="#4A4A4A" tabindex="0"></div></span></div><div style="width:16px;height:16px;margin:0 10px 10px 0"><span><div style="background:#9B9B9B;height:100%;width:100%;cursor:pointer;position:relative;outline:none;-ms-border-radius:3px;-moz-border-radius:3px;-o-border-radius:3px;-webkit-border-radius:3px;border-radius:3px;-ms-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-moz-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-o-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-webkit-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);box-shadow:inset 0 0 0 1px rgba(0,0,0,.15)" title="#9B9B9B" tabindex="0"></div></span></div><div style="width:16px;height:16px;margin:0 10px 10px 0"><span><div style="background:#FFFFFF;height:100%;width:100%;cursor:pointer;position:relative;outline:none;-ms-border-radius:3px;-moz-border-radius:3px;-o-border-radius:3px;-webkit-border-radius:3px;border-radius:3px;-ms-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-moz-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-o-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);-webkit-box-shadow:inset 0 0 0 1px rgba(0,0,0,.15);box-shadow:inset 0 0 0 1px rgba(0,0,0,.15)" title="#FFFFFF" tabindex="0"></div></span></div></div></div></div>
</content:encoded>
</item>
```

</details>

#### Feedly screenshot

![image](https://user-images.githubusercontent.com/551247/51231574-d54ccb00-1918-11e9-85be-7ef11a2088b8.png)

**Comment by ChristopherBiscardi** [comment]
I think your pasted snippet was using the `excerpt` and not the `html`? so this may be a different issue with how the text extraction for excerpts work

**Comment by ChristopherBiscardi** [comment]
Going to close this since the React issue is fixed in the latest stable release (0.3.4), but we could open a new issue for react components in the excerpt.

**Comment by stephanschubert** [comment]
You're right, I was using `excerpt`—I will open another ticket.
